airbyte-integrations/connectors/destination-snowflake/src/main/resources/spec.json

```diff
@@ -20,7 +20,7 @@
     "additionalProperties": true,
     "properties": {
       "host": {
-        "description": "The host domain of the snowflake instance (must include the account, region, cloud environment, and end with snowflakecomputing.com).",
+        "description": "The host domain of the snowflake instance (must include the account, region, cloud environment, and end with snowflakecomputing.com). The account identifier differs depending on your cloud region, be sure to verify with Snowflake's documentation.",
         "examples": ["accountname.us-east-2.aws.snowflakecomputing.com"],
```

Comment:
@lucaswiley what do you think adding another example here `xy12345.snowflakecomputing.com` without the region?

Reply:
Done